clients/netcore/Tributech.Dsk.Api.Clients.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
 
     <PackageId>Tributech.Dsk.Api.Clients</PackageId>
-    <Version>1.9.0</Version>
+    <Version>1.9.1-rc1</Version>
```

Comment:
We always started with `rc0` so we should stick to it.
see https://www.nuget.org/packages/Tributech.Dsk.Api.Clients/